bodhi/server/models.py

```diff
@@ -151,10 +152,8 @@ def __iter__(cls):
         return iter(cls._reg.values())
 
 
-class DeclEnum(object):
+class DeclEnum(six.with_metaclass(EnumMeta, object)):
     """Declarative enumeration."""
-
```

Comment:
The test failure is due to removing this blank line:

```
f27	bodhi/server/models.py:155 in public class `DeclEnum`:
f27	        D204: 1 blank line required after class docstring (found 0)
```

It's an easy fix, so I'll take care of it for you.